docs/genai/deployment/deploy-serving-graph-as-job.ipynb

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        run_object = self.project.run_function(job, inputs=inputs, local=local)
</pre><p><code>local</code> is undefined here. The example should probably be for either <code>local=True</code> or <code>local=False</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8900/discussion/#comment-2521623193'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>